.circleci/config.yml

```diff
@@ -115,6 +116,27 @@ jobs:
       - run: make check
       - run: make checkscripts
 
+  build-binaries:
+    executor: go
+    environment:
+      <<: *COMMON_ENVS
+      GOPATH: /go
+      # TODO: add ui tag here
+      GO_TAGS: "codegen_generated"
+    steps:
+      - checkout
+      - run: apt-get update; apt-get install -y sudo unzip zip
+      - run: make deps
+      - install-protoc
+      - run: sudo -E PATH="$GOPATH/bin:/usr/local/go/bin:$PATH" make generate-structs
+      - run: make pkg/windows_amd64.zip pkg/linux_amd64.zip
+      - store_artifacts:
+          path: pkg/windows_amd64.zip
+          destination: /builds/nomad_windows_amd64.zip
+      - store_artifacts:
+          path: pkg/linux_amd64.tar.gz
+          destination: /builds/nomad_linux_amd64.tar.gz
```

Comment:
FYI - I updated this to `.zip` in my rebase in https://github.com/hashicorp/nomad/pull/6745/files#diff-9b8261215d9e3483fd430aa6c6a6d0f1R15-R17 e.g. https://circleci.com/gh/hashicorp/nomad/21610#artifacts/containers/0